.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -31,42 +38,61 @@ body:
+      Describe your issue briefly.
+
+      You can include images or videos with drag and drop, and format code
+      blocks or logs with <code>```</code> tags, on separate lines before and
+      after the text. (Use <code>```gdscript</code> to add GDScript syntax
+      highlighting.)
+
+      > [!IMPORTANT]
+      >  Please do **not** add code examples or error messages as screenshots
   validations:
     required: true
 
 - type: textarea
   attributes:
     label: Steps to reproduce
     description: |
-      List of steps or sample code that reproduces the issue. Having reproducible issues is a prerequisite for contributors to be able to solve them.
-      If you include a minimal reproduction project below, you can detail how to use it here.
+      List of steps or sample code that reproduces the issue. Contributors need
+      to reproduce issues in order to fix them.
   validations:
     required: true
 
 - type: textarea
   attributes:
     label: Minimal reproduction project (MRP)
     description: |
-      - A small Godot project which reproduces the issue, with no unnecessary files included. Be sure to not include the `.godot` folder in the archive (but keep `project.godot`).
-      - Having an MRP is very important for contributors to be able to reproduce the bug in the same way that you are experiencing it. When testing a potential fix for the issue, contributors will use the MRP to validate that the fix is working as intended.
-      - If the reproduction steps are not project dependent (e.g. the bug is visible in a brand new project), you can write "N/A" in the field.
-      - Drag and drop a ZIP archive to upload it (max 10 MB). **Do not select another field until the project is done uploading.**
-      - **Note for C# users:** If your issue is *not* C#-specific, please upload a minimal reproduction project written in GDScript. This will make it easier for contributors to reproduce the issue locally as not everyone has a .NET setup available.
+      Contributors will use the MRP to validate that the fix is working as
+      intended.
+      - Drag and drop a ZIP archive containing a small Godot project which
+        reproduces the issue to upload it (max 10 MB). **Do not select another
+        field until the project is done uploading.**
+      - Make sure that the project contains the minimal amount of assets or
+        code.
+      - Be sure to not include the `.godot` folder in the archive (but keep
+        `project.godot`).
+      - We encourage the use of GDScript.
+      - Detail how to use your MRP if needed.
+
+      ---
+
+      If you are **absolutely sure** that the reproduction steps are not
+      project-dependent (the bug can be also seen in a brand new project), you
+      can write "N/A" in the field.
```

Comment:
Can we include specific example clauses? I believe that N/A is allowed “only” in the following 4 main cases:

- If the problem does not depend on a specific Node configuration
- If the problem is not dependent on a specific Resource
- Visual or usability issues in the editor (In this case a screenshot or video is required instead)
- Problems outside the Godot Editor related to builds

In all other cases, I believe that MRP should be “required” basically.

Reply:
> Can we include specific example clauses?

The idea was to be terse as possible.

> we must avoid the possibility that the author will expand on that sentence to lazy side and write only N/A.

I think the **absolutely sure** part is the only bold part in the whole issue template. It gives the seriousness of our inquiry.

Reply:
Since the criteria for making that determination are not expressed, it makes little sense for it to be in bold. So I would argue that examples need to be included because they could lead to wrong "absolutely sure".

Reply:
The list I said above is a white list of the examples, but conversely, it would be okay to note a bleck list of examples, as follows:
- If the issue occurs with something imported resources
- If the issue occurs with a project which has several Nodes or Resources